### DISCUSSION TITLE
社区已迁移到 GitHub Discussion

### DISCUSSION THREAD

**Discussion by Redbeanw44602** [PenUniverse/PenMods#172] [发布专区]
为什么突然迁移呢...其实是因为我一不小心把原社区的数据全删了，也没备份。本来想折腾数据恢复的，但是思来想去——本来用专门的服务器来建 PenMods 的社区，就是拿大炮打蚊子的行为。说实话，PenMods项目在我心目中的优先级非常之不高... 所以有时候对这个项目就有些缺乏耐心（以至于连备份都懒得搞）...
给各位造成不便还望谅解，数据都没了，怎么办呢？
好在，有一说一，社区里有价值的东西并不是很多（有几位朋友写了关于New Bing代理和巧用错题本一类的技巧，可惜了），这也反映出词典笔社区的荒索哈哈，进一步证明了大炮打蚊子的不必要性... 总之 GitHub Discussion 也挺好（鉴定为比QQ群强），也不比原来的社区功能差，就这样吧，玩笔图一乐，要什么飞机，哈哈。

**Comment by Orange-Black** [comment]
大佬问一下，原来的论坛用的是什么论坛系统？

**Reply by Redbeanw44602** [reply]
> 大佬问一下，原来的论坛用的是什么论坛系统？

NodeBB

**Reply by Orange-Black** [reply]
> > 大佬问一下，原来的论坛用的是什么论坛系统？
> 
> NodeBB

好的，谢谢！
